### PR TITLE
Replace watch with new implementation, gain UTF-8

### PR DESCRIPTION
**Pull request by Mno-hime**

### CODE REVIEW

File: components/sysutils/procps/Makefile [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+PREFERRED_BITS=		64
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		procps
+COMPONENT_VERSION=	3.3.15
+COMPONENT_SUMMARY=	Linux ps utilities
+COMPONENT_PROJECT_URL=	https://gitlab.com/procps-ng/procps
+COMPONENT_FMRI=		utilities/procps
```
Comment (pyhalov) [inline components/sysutils/procps/Makefile]: at least utility/procps to be consistent, perhaps, better keep existing name as it just the same watch?

**Comment by pyhalov** [comment]
You forgot to remove actual watch component. Perhaps, preserve the same fmri and component location, just switch source?

**Comment by Mno-hime** [comment]
OK. Just replaced the component with new source.

**Comment by pyhalov** [comment]
Hey, it doesn't build. Likely you forgot patches.

**Comment by Mno-hime** [comment]
Yup, they are here now.

**Pull request merged**